### PR TITLE
Multi step scheduler

### PR DESCRIPTION
**Pull request by vict0rsch**
Change the learning rate scheduler

### CODE REVIEW

File: omnigan/trainer.py [hunk 1]
```diff
@@ -248,17 +248,19 @@ def setup(self):
...
         # Get different optimizers for each task (different learning rates)
         self.g_opt, self.g_scheduler = get_optimizer(
-            self.G, self.opts.gen.opt, tasks=self.opts.tasks
+            self.G, self.opts.gen.opt, self.opts.tasks, self.opts.train.epochs
         )
 
         if get_num_params(self.D) > 0:
-            self.d_opt, self.d_scheduler = get_optimizer(self.D, self.opts.dis.opt)
+            self.d_opt, self.d_scheduler = get_optimizer(
+                self.D, self.opts.dis.opt, self.opts.train.epochs
```
Comment (alexrey88) [inline omnigan/trainer.py:256]: Wouldn't it put tasks=self.opts.train.epochs in the function instead of iterations=self.opts.train.epochs like you want?

Comment (vict0rsch) [reply omnigan/trainer.py:256]: you're right it's a typo thanks!

File: omnigan/utils.py [hunk 2]
```diff
@@ -150,6 +150,10 @@ def get_git_revision_hash():
         return str(e)
 
 
+def kill_job(id):
```
Comment (melisandeteng) [inline omnigan/utils.py:153]: very useful indeed 😄

File: shared/trainer/defaults.yaml [hunk 3]
```diff
@@ -7,6 +7,8 @@ load_paths:
 # -----  Tasks  -----
 # -------------------
 tasks: [d, s, m, p] # [p] [m, s, d]
+# Notes:
+# if using `s` be sure to use `train_r_full_pl.json` and `val_r_full_pl.json`
```
Comment (melisandeteng) [inline shared/trainer/defaults.yaml:11]: now you can leave that out and put train_r_full.json and val_r_full.json

**Review by melisandeteng — APPROVED** [review]
